modules/ui/photoviewer.js

```diff
@@ -146,15 +150,18 @@ export function uiPhotoviewer(context) {
                 button.remove();
             }
 
-            function buttonShowHide() {
-                const activeImage = services.mapillary.getActiveImage();
+            function buttonShowHide(tagName) {
+                const activeImage = services[tagName].getActiveImage();
 
                 const graph = context.graph();
                 const entities = context.selectedIDs()
                     .map(id => graph.entity(id));
 
-                if (entities.map(entity => entity.tags.mapillary)
-                    .every(value => value === activeImage?.id)) {
+                const entitiesWithTag = entities.map(entity => entity.tags[tagName] ? entity.tags[tagName] : undefined).filter(e => e);
```

Comment:
That should be the same but shorter, right?

```suggestion
                const entitiesWithTag = entities.map(entity => entity.tags[tagName]).filter(Boolean);
```

Reply:
nice, thanks